### PR TITLE
feat: new menu structure

### PR DESCRIPTION
**Pull request by henryiii**
> "when everyone is super, no one will be."
> 
> -- Syndrome, The Incredibles

This is the new menu structure I've been working on. It's more organized, avoids larger-than-the-screen menus on mobile, and keeps the growing number of pages logically sorted. We had too many top level items with things that usually are in About or similar places. I've left "news" as a top level item, and all the others are within menus. To counteract the slightly higher discoverability of a few pages, I've added a few key pages to the front page, where they are highly visible, but don't clog up the menu system.

I've also restyled the note blocks, improved cross links in a few places, and updated a bit of text. I removed the two badges - There's already a GitHub link, and there's now a link to Getting in touch that includes Gitter.

You can see the new menu (when expanded) here:

Screen Shot 2022-05-13 at 2 33 35 PM

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,15 +6,21 @@ nav_order: 1
...
-for information on developing Python packages!
+analysis in Python. [Read more →]({{site.baseurl}}{% link pages/about/index.md %})
+
+New users can start with our
+[user pages]({{site.baseurl}}{% link pages/user/index.md %}). See our
+[developer pages]({{site.baseurl}}{% link pages/developers/index.md %})
+for information on developing Python packages.
+
+<div markdown="1" class="text-center text-delta fs-3">
+[News]({{site.baseurl}}{% link pages/news.md %}) •
```
Comment (henryiii) [inline index.md:19]:
```suggestion
[News]({{site.baseurl}}{% link pages/news.md %}) •
[Tutorial](https://hsf-training.github.io/hsf-training-scikit-hep-webpage/) •
```

File: pages/user/resources.md [hunk 2]
```diff
@@ -13,7 +14,10 @@ or related to, the Scikit-HEP project.
 
 ## Scikit-HEP tutorials
 
-See the [Jupyter Book](https://scikit-hep.org/scikit-hep-tutorials/).
+
+* [HSF Scikit-HEP tutorial](https://hsf-training.github.io/hsf-training-scikit-hep-webpage/)
+* [WIP Scikit-HEP Jupyter Book](https://scikit-hep.org/scikit-hep-tutorials/).
```
Comment (henryiii) [inline pages/user/resources.md:19]:
```suggestion
* [WIP Scikit-HEP Jupyter Book](https://scikit-hep.org/scikit-hep-tutorials/)
```

**Comment by jpivarski** [comment]
Packing everything under the same number of folders gives it a better sense of organization. I'm in favor!

File: pages/user/resources.md [hunk 3]
```diff
@@ -13,7 +14,9 @@ or related to, the Scikit-HEP project.
 
 ## Scikit-HEP tutorials
 
-See the [Jupyter Book](https://scikit-hep.org/scikit-hep-tutorials/).
+- [HSF Scikit-HEP tutorial](https://hsf-training.github.io/hsf-training-scikit-hep-webpage/)
+- [WIP Scikit-HEP Jupyter Book](https://scikit-hep.org/scikit-hep-tutorials/).
```
Comment (henryiii) [inline pages/user/resources.md:18]:
```suggestion
- [WIP Scikit-HEP Jupyter Book](https://scikit-hep.org/scikit-hep-tutorials/)
```

**Comment by henryiii** [comment]
It's been open for a week, and I've got one `+1`, and zero `-1`'s, so will merge later today unless anyone shouts out. We can always change again if needed.

**Pull request merged**